src/Model/Entity/Transcription.php

```diff
@@ -140,4 +154,80 @@ private function getSourceScript($sourceLang) {
         }
         return false;
     }
+
+    protected function _getHtml() {
+        if (!$this->sentence) {
+            return;
+        }
+
+        $lang = $this->sentence['lang'];
```

Comment:
Just in case, you should also check for `$this->sentence['lang']`. And it’s clearer to use `$this->sentence->lang`. So something like:
```php
if (!$this->sentence || !$this->sentence->lang) {
    return;
}
```

Reply:
And also `$this->script` and `$this->text`.